pepper-apis/src/main/java/org/broadinstitute/ddp/service/WorkflowService.java

```diff
@@ -172,7 +172,7 @@ private String getStudyName(Handle handle, String userGuid, String studyGuid) {
         if (preferredLanguageCode != null) {
             List<StudyI18nDto> studyTranslations = translationDao.findTranslationsByStudyId(study.getId());
             for (StudyI18nDto dto : studyTranslations) {
-                if (dto.getLanguageCode() != null && dto.getLanguageCode() == preferredLanguageCode) {
+                if (dto.getLanguageCode() != null && dto.getLanguageCode().equals(preferredLanguageCode)) {
```

Comment:
Found a bug: we should compare strings with `equals()`.

Reply:
Nice catch! For anyone wondering why- when dealing with objects, `==` is reference equality (do both sides point to the same location in memory), where `equals(...)` is object equality (do the objects have the same content).